### DISCUSSION TITLE
OP network to ERC20 USDT lost

### DISCUSSION THREAD

**Discussion by DilaG9** [ethereum-optimism/developers#534] [General]
### Discussion Topic

Dear sir/madam,
I’m  withdraw 1500.99 USDT OP network erc20 / pionex to coinbase exchange.But unfortunately coinbase not supported OP to erc20 .they said my assets hold because of wrong network,when they supported OP it will happens.But I’m in really critical situation,so please be kindly gide me to recover my assets back to pionex. once again thanks for your cooperation.       
Best Regards

**Comment by imtipi** [comment]
need to contact coinbase support because receiver address is control by coinbase

**Reply by DilaG9** [reply]
no they said they can not do anyting when the OP is supported ERC20 it will happen but they can not grantee.is there any possibility to revised through OP network?

**Reply by imtipi** [reply]
can't,have to wait coinbase to support

**Reply by DilaG9** [reply]
ok

**Reply by imtipi** [reply]
and coinbase has a function called asset recovery,can you try is it working for op mainnet usdt?if not I will report this to team(although also no grantee)
https://help.coinbase.com/en/coinbase/trading-and-funding/sending-or-receiving-cryptocurrency/lost-assets

**Reply by DilaG9** [reply]
I followed already but its not mention OP network

**Comment by DilaG9** [comment]
I already followed the recovery process but there is only ERC20/BNB/Matic
only no OP network.Coinbase not supported anymore, they said when the op
network add after to coinbase, recover my money But no more guaranteed.If i
can possible to revise asset by direct contact OP network.im also
struggling whats next ?

**Reply by hash1go** [reply]
I'm sorry to hear that.
If Coinbase does not support the recovery, there is no way to retrieve your tokens.
